src/hotspot/share/gc/g1/g1ConcurrentMark.cpp

```diff
@@ -586,24 +586,49 @@ class G1ClearBitMapTask : public AbstractGangTask {
   private:
     G1CMBitMap* _bitmap;
     G1ConcurrentMark* _cm;
+
+  bool has_aborted() {
+    if (_cm != NULL) {
```

Comment:
Maybe put the predicate `_cm != nullptr` in an extra method, something like `is_clear_concurrent()` (or a better name) would improve readability...